graphistry/feature_utils.py

```diff
@@ -167,7 +168,7 @@ def resolve_feature_engine(
 
 def resolve_y(df: Optional[pd.DataFrame], y: YSymbolic) -> pd.DataFrame:
 
-    if isinstance(y, pd.DataFrame):
+    if isinstance(y, pd.DataFrame) or isinstance(y, cudf.DataFrame):
```

Comment:
a trick to avoid an import here may be to use an introspection routine like `'cudf.core.dataframe' in str(getmodule(o))`